collection.php

```diff
@@ -9,22 +9,39 @@
 ?>
 
 <!DOCTYPE html>
-    <body>
-        <main>
-            <h1>Harry Potter Characters</h1>
-            <div>
-
-                <?php
 
-                foreach ($hpcharacters as $character) {
-                    echo '<h3>' . $character["name"] . '</h3>';
+<head>
+    <link rel="stylesheet" type="text/css" href="collection.css"/>
+    <title>HP Collection</title>
+</head>
+<body>
+    <main>
+        <div>
+            <h1>Harry Potter Characters</h1>
+        </div>
+        <div class="characters">
+            <?php
+            foreach ($hpcharacters as $character) {
+                if ($character["house"] === 'Gryffindor') {
```

Comment:
You may be able to put these in functions 
for unit tests ( if yo u want to ofc)